### DISCUSSION TITLE
can renovate update the base image if it has hosted on ECR?

### DISCUSSION THREAD

**Discussion by n7z2** [renovatebot/renovate#20385] [Request Help]
### How are you running Renovate?

Mend Renovate hosted app on github.com

### If you're self-hosting Renovate, tell us what version of Renovate you run.

34.29.2

### If you're self-hosting Renovate, select which platform you are using.

gitlab.com

### Was this something which used to work for you, and then stopped?

It used to work, and then stopped

### Wanted end result.

If you are running renovate on a gitlab runner, and like how it updates the base image of the dockerfile. Can renovate do the same if the base image of the docker file is an private based ECR URL, and that ecr hosted image contains a base image from dockerhub like java:11. Do I need to give the gitlab runner extra AWS permissions or is there something I am missing.

### What you tried so far.

I tested this locally and it could not read the private ECR repos.

### Relevant debug logs

<details><summary>Logs</summary>

```
Copy/paste the relevant log(s) here, between the starting and ending backticks
```

</details>

**Comment by rarkins** [comment]
> It used to work, and then stopped

In which version? Please correct your submission if this is in error
